### ISSUE TITLE
The value of expressions being monitored sometimes doesn't change.

### ISSUE THREAD

**Issue by iosmanthus** [cs01/gdbgui#182]
##### Describe the issue
First of all, I have `int foo = 1;`
if I just monitor the value of an ordinary variable, it changes properly(e.g. `foo`). But when I try to monitor more complex expressions(e.g. `(void *)foo`), the vaule doesn't change at all.

![screenshot from 2018-04-07 09-00-49](https://user-images.githubusercontent.com/16307070/38449477-40bcdd5a-3a42-11e8-879e-cd25f124e982.png)

##### Describe your environment
* Operating system and version:
`ArchLinux up to date`
* gdbgui version (`gdbgui -v`):
`0.11.2.0`
* gdb version (`gdb -v`):
```
GNU gdb (GDB) 8.1
Copyright (C) 2018 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.  Type "show copying"
and "show warranty" for details.
This GDB was configured as "x86_64-pc-linux-gnu".
Type "show configuration" for configuration details.
For bug reporting instructions, please see:
<http://www.gnu.org/software/gdb/bugs/>.
Find the GDB manual and other documentation resources online at:
<http://www.gnu.org/software/gdb/documentation/>.
For help, type "help".
Type "apropos word" to search for commands related to "word".
```
* browser:
`Google Chrome 65.0.3325.181 `
* python packages (`pip freeze`):
```
aniso8601==3.0.0
appdirs==1.4.3
astroid==1.6.1
autopep8==1.3.4
blessed==1.14.2
chardet==3.0.4
click==6.7
colorama==0.3.9
decorator==4.2.1
Flask==0.12.2
Flask-Compress==1.4.0
Flask-RESTful==0.3.6
Flask-SocketIO==2.9.6
gdbgui==0.11.2.0
gevent==1.2.2
greenlet==0.4.13
idna==2.6
isort==4.3.4
itsdangerous==0.24
Jinja2==2.10
lazy-object-proxy==1.3.1
lit==0.6.0.dev0
louis==3.4.0
MarkupSafe==1.0
mccabe==0.6.1
packaging==16.8
psutil==5.4.3
pycodestyle==2.3.1
pygdbmi==0.8.2.0
Pygments==2.2.0
pygobject==3.28.2
pylint==1.8.2
pyparsing==2.2.0
pysigset==0.3.2
pyte==0.7.0
python-engineio==2.0.4
python-socketio==1.9.0
pytz==2018.3
pyxdg==0.25
PyYAML==3.12
requests==2.18.4
requests-unixsocket==0.1.5
scruffington==0.3.7
shadowsocks==2.9.1
six==1.11.0
team==1.0
thefuck==3.25
urllib3==1.22
wcwidth==0.1.7
Werkzeug==0.14.1
wrapt==1.10.11
ydcv==0.5
```

**Comment by cs01** [comment]
Thanks for filing an issue, I will check this out and get back to you.

**Comment by cs01** [comment]
Great catch, this was a bug. It should be fixed in v0.11.3.0.
